### PR TITLE
Remove support for GCE Alpha Disk Insert API

### PR DESCRIPTION
**Pull request by amacaskill**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup

> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
This is a follow up PR to [PR 1666](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1666), which Migrated Hyperdisk/CHD/StoragePools features to the GCE v1 disk API. Now that no features are using the GCE alpha disk API, this PR removes support for Alpha Disks. It also adds v1 support to CloudDisk (/gce-cloud-provider/compute/cloud-disk.go) for the Hyperdisk/CHD/StoragePools features (d *CloudDisk) GetEnableConfidentialCompute()` 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Remove support for GCE Alpha Disk Insert API.
```

### CODE REVIEW

File: pkg/gce-cloud-provider/compute/cloud-disk.go [hunk 1]
```diff
@@ -134,8 +115,6 @@ func (d *CloudDisk) GetPDType() string {
 		pdType = d.disk.Type
 	case d.betaDisk != nil:
 		pdType = d.betaDisk.Type
-	case d.alphaDisk != nil:
```
Comment (sunnylovestiramisu) [inline pkg/gce-cloud-provider/compute/cloud-disk.go:137]: How will this impact the endpoint in TPC?

File: pkg/gce-cloud-provider/compute/cloud-disk.go [hunk 2]
```diff
@@ -18,15 +18,13 @@ import (
 	"strings"
 
 	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/meta"
-	computealpha "google.golang.org/api/compute/v0.alpha"
 	computebeta "google.golang.org/api/compute/v0.beta"
 	computev1 "google.golang.org/api/compute/v1"
 )
 
 type CloudDisk struct {
```
Comment (sunnylovestiramisu) [inline pkg/gce-cloud-provider/compute/cloud-disk.go:25]: Can we add a comment here like "if you want to add alpha API please refer to this PR" ?

Comment (amacaskill) [reply pkg/gce-cloud-provider/compute/cloud-disk.go:25]: I added this to cloud provider interface which is what stores the alpha service. I didn;t think we needed it in both places. What do you think?

```
// CloudProvider only supports GCE v1/beta Disk APIs. See
// https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1524
// for how to add GCE alpha Disk support.
```

Comment (sunnylovestiramisu) [reply pkg/gce-cloud-provider/compute/cloud-disk.go:25]: Oh I missed this, yeah one place is fine.

File: pkg/gce-cloud-provider/compute/cloud-disk.go (hunk 1 above)
Comment (sunnylovestiramisu) [reply pkg/gce-cloud-provider/compute/cloud-disk.go:137]: I think just that in TPC it will also only use beta/v1 which should not be a problem

**Pull request merged**